proj/settings.py

```diff
@@ -120,11 +120,13 @@
 
 LOGIN_REDIRECT_URL = '/'
 
+SENDGRID_API_KEY = os.getenv('SENDGRID_API_KEY')
+
 EMAIL_USE_TLS = True
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_HOST_USER = 'stachackweek@gmail.com'
-EMAIL_HOST_PASSWORD = 'hackstac'
+EMAIL_HOST = 'smtp.sendgrid.com'
+EMAIL_HOST_USER = 'apikey'
+EMAIL_HOST_PASSWORD = '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'
```

Comment:
Ideally, you should use the environment variable you got in Line 123 rather than adding the API key directly in the code as others can also get access to your email account using the key. We will still be using this method for development purpose but we would have to change this afterwards.